portal/tests/pageObjects/portal/teach/dashboard_page_new.py

```diff
@@ -35,10 +35,83 @@
 # program; modified versions of the program must be marked as such and not
 # identified as the original program.
 from teach_base_page_new import TeachBasePage
+from selenium.webdriver.support.ui import Select
+
+import time
 
 
 class TeachDashboardPage(TeachBasePage):
     def __init__(self, browser):
         super(TeachDashboardPage, self).__init__(browser)
 
         assert self.on_correct_page('teach_dashboard_page_new')
+
+    def check_organisation_details(self, details):
+        correct = True
+
+        first_field = details.items()[0][0]
+        self.wait_for_element_by_id('id_' + first_field)
+
+        for field, value in details.items():
+            correct &= (self.browser.find_element_by_id('id_' + field).get_attribute('value') == value)
+
+        return correct
+
+    def change_organisation_details(self, details):
+        for field, value in details.items():
+            self.browser.find_element_by_id('id_' + field).clear()
+            self.browser.find_element_by_id('id_' + field).send_keys(value)
+
+        self.browser.find_element_by_id('update_details_button').click()
+        return self
+
+    def has_edit_failed(self):
+        self.wait_for_element_by_id('edit_form')
+        errorlist = self.browser.find_element_by_id('edit_form').find_element_by_class_name('errorlist').text
+        error = 'There is already a school or club registered with that name and postcode'
```

Comment:
with localisation and things that won't work well / be very tricky, is it possible to use some kind of id instead?